### ISSUE TITLE
Types for typescript are not exported for @tensorflow/tfjs-layers

### ISSUE THREAD

**Issue by mctrafik** [tensorflow/tfjs#6843]
I'm importing @tensorflow/tfjs@3.18 and most of the api subpackages have corresponding `*.d.ts` files, but nothing in the `-layers` package. JS only. I've checked for `@types/tensorflow__tfjs-layers` and they haven't been generated either. So I think this is an oversight somewhere. Makes it impossible to do classification in typescript. Please fix.

**Describe the current behavior**

`tf.sequential` is undefined
No types declared for `@tensorflow/tfjs-layers`.

**Describe the expected behavior**

I expect the types to be exported so typescript applications could compile.

**Comment by mattsoulanille** [comment]
Thanks for the bug report! The types seem to be present on 3.18.0 and 3.20.0 when I check. I can compile the following tiny project and get tsserver editor autocompletion. Does this work on your computer?

tsconfig.json generated by `tsc --init` with comments removed
```json
{
  "compilerOptions": {
    "target": "es2016",
    "module": "commonjs",
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
    "strict": true,
    "skipLibCheck": true
  }
}
```

package.json
```json
{
  "name": "test-tfjs",
  "version": "1.0.0",
  "main": "index.js",
  "license": "Apache-2.0",
  "devDependencies": {
    "typescript": "^4.8.3"
  },
  "dependencies": {
    "@tensorflow/tfjs": "3.20.0"
  }
}
```

index.ts
```typescript
import * as tf from '@tensorflow/tfjs';

console.log(tf.sequential);
```

`tsc -p tsconfig.json && node index.js`

**Comment by mctrafik** [comment]
Interesting. Here's what I have:

Here's what I see in my node_modules for @tensorflow/tfjs-core (notice all the *.d.ts files)
![image](https://user-images.githubusercontent.com/1694347/191619120-3e6fbfe1-bcea-4c6d-848f-bbad7d935a20.png)

But for tfjs-layers:

![image](https://user-images.githubusercontent.com/1694347/191619347-6c45a8bd-ae94-45d2-96a3-3c1eedbb2703.png)

Since you mentioned that it worked for you, I've cleaned my cache, removed the package, and reinstalled just `@tensorflow/tfjs-layers` and now I got the types! Thank you.
